packages/amplication-data-service-generator/src/tests/__snapshots__/create-data-service-custom-path.spec.ts.snap

```diff
@@ -1648,11 +1648,12 @@ export const CustomerShow = (props: ShowProps): React.ReactElement => {
 };
 ",
   "test-ui/src/customer/CustomerTitle.ts": "import { Customer as TCustomer } from \\"../api/customer/Customer\\";
+// declare class ENTITY_TITLE_RETURN_TYPE {}
```

Comment:
Please remove this declare class

Reply:
Yep. I forgot to update the snapshot after trying to implement a dynamic return type 😅